tests/pytests/integration/runners/test_nacl.py

```diff
@@ -0,0 +1,162 @@
+"""
+Tests for the nacl runner
+"""
+
+import pytest
+
+import salt.config
+import salt.utils.stringutils
+from tests.support.mock import patch
+
+pytest.importorskip("libnacl")
+
```

Comment:
```suggestion
pytest.importorskip("nacl")

```

Sorry, wrong import. It needs to be whatever we import in `salt.utils.nacl`

Reply:
got importorskip working